test/humanizer.coffee

```diff
@@ -31,6 +31,22 @@ describe "humanizer", ->
     assert.equal h(ms("6h")), "0.25 days"
     assert.equal h(ms("7d")), "7 days"
 
+  it "can overwrite the unit measures in the initializer", ->
+    h = humanizer
+      unitMeasures:
```

Comment:
Are these measurements different from the defaults? If not, could you make them different?